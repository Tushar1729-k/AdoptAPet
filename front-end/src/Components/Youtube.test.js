import React from 'react';
import YoutubePlayer from "./Youtube";
import renderer from 'react-test-renderer';

test("Youtube video properly rendered", () => {
    const component = renderer.create(
        <YoutubePlayer searchQuery={"Domestic Short Hair"} />,
      );
      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
})