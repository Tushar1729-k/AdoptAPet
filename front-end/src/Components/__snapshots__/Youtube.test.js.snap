// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Youtube video properly rendered 1`] = `
<div>
  <button
    className="btn btn-primary"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    Youtube
  </button>
</div>
`;
